articles/iot-dps/quick-setup-auto-provision-cli.md

```diff
@@ -42,7 +42,7 @@ az group create --name my-sample-resource-group --location westus
 
 ## Create an IoT hub
 
-Create an IoT IoT hub with the [az iot hub create](/cli/azure/iot/hub#az-iot-hub-create) command. 
+Create an IoT hub with the [az iot hub create](/cli/azure/iot/hub#az-iot-hub-create) command
```

Comment:
Oops, looks like I dropped the period when doing the online conflict resolution. I'll try and add it back when I'm at another machine.

Reply:
I added the period. Thank you